src/commands/play.rs

```diff
@@ -75,40 +79,28 @@ pub async fn _play(
             EnqueueType::URI | EnqueueType::SEARCH => match flag {
                 PlayFlag::PLAYTOP => {
                     let track = queue.get(1).unwrap();
-                    let embed = create_queued_embed("Added to top", track, estimated_time).await;
+                    let embed =
+                        create_queued_embed("⌛  Added to top", track, estimated_time).await;
```

Comment:
In retrospect, using ⌛ looks like it's still loading. Maybe a 👍 or ✅ would be more intuitive.